dashboard/src/t5gweb/templates/ui/stats.html

```diff
@@ -205,7 +205,7 @@ <h2>Breakdowns:</h2>
 
             <table class="table table-bordered table-hover table-responsive mt-5 w-100 display" id="engineers">
                 <thead>
-                    <tr><th class="text-center">Engineer</th><th class="text-center">Number of Cases</th></tr>
+                    <tr><th class="text-center">Engineer</th><th class="text-center">Number of Open Cases</th></tr>
```

Comment:
Since there might have been some confusion over which engineers this is referring to, maybe we could change the title of the column to `Field Engineer` instead of `Engineer` while we're already changing this part of the code

Reply:
good call! fixed. also noticed put an "Open" in the wrong spot and fixed it